proxy/proxy.go

```diff
@@ -0,0 +1,19 @@
+package proxy
+
+import (
+	"os"
+	"os/exec"
+	"syscall"
+
+	"github.com/pkg/errors"
+)
+
+func Exec(command string, args []string) error {
+	agent, err := exec.LookPath("picard")
+	if err != nil {
+		return errors.Wrap(err, "Could not find `picard` executable on $PATH; please ensure that build-agent is installed")
```

Comment:
The error returned by `exec.LookPath` (https://golang.org/src/os/exec/lp_unix.go#L34, https://golang.org/src/os/exec/exec.go#L44) will be something like 

```
exec: picard: executable not found in $PATH
```

So in the wrap message we probably don't want to duplicate those details. I think we could say something like `errors.Wrapf(err, "failed to proxy command %s", command)`. 

If we only expect this to work inside of a circleCI job we could explain that as well.